remappings.txt

```diff
@@ -1,5 +1,6 @@
 ds-test/=packages/protocol/lib/celo-foundry/lib/forge-std/lib/ds-test/src/
 forge-std/=packages/protocol/lib/celo-foundry/lib/forge-std/src/
+forge-std-8/=packages/protocol/lib/celo-foundry-8/lib/forge-std/src/
```

Comment:
I wonder why we had this package if it was previously unused

Reply:
This is used in `foundry.toml`. It was added to test GasPriceMinimum